docs/integrations/containers-orchestration/vmware-tanzu-application-service.md

```diff
@@ -0,0 +1,119 @@
+---
+id: vmware-tanzu-application-service
+title: VMware Tanzu Application Service
+sidebar_label: VMware Tanzu Application Service
+description: This documentation describes the Sumo Logic Nozzle for VMware Tanzu tile.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/containers-orchestration/vmware-tanzu.png')} alt="VMware Tanzu Application Service" width="40" />
+
+The Sumo Logic Nozzle for [VMware Tanzu Firehose](https://docs.vmware.com/en/Tile-Developer-Guide/3.0/tile-dev-guide/nozzle.html) allows operations teams and app owners to monitor and troubleshoot the VMware Tanzu platform and apps in real time, without having to install a Sumo Logic agent in their Cloud Foundry environment.
+
+The nozzle filters out messages per your tile configuration and securely sends those logs and metrics to an HTTP API in the Sumo Logic SaaS service. The tile allows you to configure batching intervals, toggle verbose logging mode, and add metadata to your logs and metrics.
+
+The only configuration you need to do on the Sumo Logic platform is to enable the HTTP API (an HTTP Source) and record the URL. After you deploy the tile, you are able to query the logs and metrics and monitor the log stream using Live Tail.
+
+The Sumo Logic nozzle is written in Go and is based on the [firehose-to-syslog](https://github.com/cloudfoundry-community/firehose-to-syslog) nozzle.
+
+Key features of Sumo Logic Nozzle for VMware Tanzu include:
+
+* View dashboards to monitor the overall health and security of your VMware Tanzu resources.
+* Drill down from alerts to see key performance metrics and audit logs for VMware Tanzu and your custom apps.
+* Store VMware Tanzu audit details in a fully secure SOC 2, HIPAA, and PCI-compliant environment.
+
+## Prerequisites
+
+Before you install the Sumo Logic Nozzle for VMware Tanzu tile, you'll need Administrative access to your VMware Tanzu deployment.
+
+### Product snapshot
```

Comment:
We can remove this product snapshot section and instead add a note

To know about compatible Tanzu Application Service versions refer the latest [release](https://github.com/SumoLogic/sumologic-cloudfoundry-nozzle/releases).